scripts/tests/configure_qwen_no_thinking.py

```
@@ -0,0 +1,129 @@
+    except FileNotFoundError:
+        print("❌ Ollama não encontrado. Certifique-se de que está instalado e no PATH")
+        return False
+    except Exception as e:
+        print(f"❌ Erro inesperado: {e}")
+        return False
+    finally:
+        # Limpar arquivo temporário
+        try:
+            os.unlink(modelfile_path)
+        except:
+            pass
```

Ruff (0.11.9)

Use `contextlib.suppress(Exception)` instead of `try`-`except`-`pass`

Replace with `contextlib.suppress(Exception)`

(SIM105)